lib/app/features/parent_side/edit_child_page.dart

```diff
@@ -89,32 +89,33 @@ class _EditChildPageState extends State<EditChildPage> {
   Future<void> _submit(XFile? localFile) async {
     if (appState == AppState.loading) return;
     if (_validateAndSaveForm()) {
-      if (localFile == null) return;
       setState(() {
         appState = AppState.loading;
       });
 
       id = uuid.v4().substring(0, 8).toUpperCase();
-      try {
-        final fileExtension = path.extension(localFile.path);
-        JHLogger.$.d(fileExtension);
+      if(localFile!=null) {
```

Comment:
Changes on this file, (if my memory is right):
- removing if statement on line 92.
- Added this if null check that deals with the image uploaded.